x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/mobile/response_duration_config.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConfigProps, DataSeries } from '../../types';
+import { FieldLabels } from '../constants';
+import { buildPhraseFilter } from '../utils';
+import { SERVICE_NAME, TRANSACTION_DURATION } from '../constants/elasticsearch_fieldnames';
+
+export function getResponseDurationLensConfig({ seriesId, indexPattern }: ConfigProps): DataSeries {
+  return {
+    id: seriesId,
+    reportType: 'service-latency',
+    defaultSeriesType: 'line',
+    seriesTypes: ['line', 'bar'],
+    xAxisColumn: {
+      sourceField: '@timestamp',
+    },
+    yAxisColumns: [
+      {
+        operationType: 'average',
+        sourceField: 'transaction.duration.us',
+        label: 'Latency',
+      },
+    ],
+    hasOperationType: true,
+    defaultFilters: [
+      'labels.net_connection_carrier_name',
+      'labels.device_model',
+      'labels.net_connection_type',
+      'host.os.platform',
+      'host.os.full',
+    ],
+    breakdowns: [
+      'labels.net_connection_carrier_name',
+      'labels.device_model',
+      'labels.net_connection_type',
+      'host.os.platform',
+      'labels.net_connection_carrier_isoCountryCode',
+    ],
+    filters: [
+      ...buildPhraseFilter('transaction.type', 'request', indexPattern),
```

Comment:
I think this filter is not applicable to transactions starting on mobile devices.
The APM server marks transactions with type 'request' only if it starts from an incoming HTTP request which is not the case for mobile transactions as they usually start from user interactions on the device. They rather are of type 'custom' or any component name if the component is set. 

How about removing this filter at all here?